app/src/main/res/layout/private_browsing_description.xml

```diff
@@ -9,7 +9,8 @@
     android:layout_height="wrap_content"
     android:layout_margin="0.5dp"
```

Comment:
I am wondering if we need to keep this 0.5dp margin that will now only apply to the vertical margins. At the same time, the margin is so small I am wondering if we can double check the old designs to see if this is needed since it's such a small margin. 

This can be a follow-up.

Reply:
I see this implemented in https://github.com/mozilla-mobile/fenix/issues/6629 with no talk about the margin value.
I'd say that after so much time (~2 years) if it wasn't in the specs then in became so in the meantime. 😂.